datahub-web-react/src/AppConfigProvider.tsx

```diff
@@ -2,7 +2,7 @@ import '@src/App.less';
 
 import React, { useEffect } from 'react';
 
-import { THIRD_PARTY_LOGGING_KEY } from '@app/analytics/analytics';
+import { SEVER_VERSION_KEY, THIRD_PARTY_LOGGING_KEY } from '@app/analytics/analytics';
```

Comment:
typo - SEVER_VERSION_KEY -> SERVER_VERSION_KEY
```suggestion
import { SERVER_VERSION_KEY, THIRD_PARTY_LOGGING_KEY } from '@app/analytics/analytics';
```